src/dvclive/live.py

```diff
@@ -525,6 +531,11 @@ def make_report(self):
     def make_dvcyaml(self):
         make_dvcyaml(self)
 
+    @catch_and_warn(Exception, logger)
+    def make_stage_template(self):
+        stage_content = get_dvc_stage_template(self)
```

Comment:
We should check either here or inside `get_dvc_stage_template` the existence of `self._dvc_repo`. We don't do it in make_dvcyaml because we don't try to access it.